starfish/core/spots/FindSpots/_base.py

```diff
@@ -0,0 +1,28 @@
+from abc import abstractmethod
+from typing import Callable, Optional, Sequence
+
+import numpy as np
+
+from starfish.core.imagestack.imagestack import ImageStack
+from starfish.core.pipeline.algorithmbase import AlgorithmBase
+from starfish.core.types import Number, SpotFindingResults
+
+
+class FindSpotsAlgorithm(metaclass=AlgorithmBase):
+
+    @abstractmethod
+    def run(self, image_stack: ImageStack,
+            reference_image: Optional[ImageStack] = None, *args) -> SpotFindingResults:
+        """Find and measure spots across rounds and channels in the provided
```

Comment:
unwrap?  I'd check your editor to make sure you're set up to 100col.  This clocks in at 92col unwrapped for me.